Description: Remove dep on alto, which is archived upstream
 (and only used in an example anyway)
Last-Update: 2023-02-10
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -32,9 +32,6 @@
 name = "perf"
 required-features = ["ogg"]

-[[example]]
-name = "player"
-required-features = ["ogg"]
 [dependencies.byteorder]
 version = "1.0"

@@ -53,8 +50,6 @@
 [dependencies.tokio-io]
 version = "0.1"
 optional = true
-[dev-dependencies.alto]
-version = "3"

 [dev-dependencies.ogg]
 version = "0.9"
